doc/documentation.md

```diff
@@ -143,31 +144,34 @@ https://api.kirjastot.fi/v4/library/<id>
 **NOTE**: `<id>` can be either `id` or `slug`.
 
 ## Query parameters
-Name            | Description
---------------- | -----------
-id              | List of library IDs.
-name            | Name of the library. Varies by `langcode`.
-slug            | URL identifier. Varies by `langcode`.
-type            | Type of library.
-status          | Filter by live status of the library. (see `/schedules`)
---              | --
-city            | List of city IDs.
-city.name       | Name of the city. Varies by `langcode`.
-consortium      | List of consortium IDs.
-consortium.name | Name of library consortium.
-service         | List of service IDs offered by libraries.
-service.name    | Search using a service's name.
---              | --
-geo.pos         | Reference point for geographic search. (`lat,lon`)
-geo.dist        | Distance from the reference point in kilometers. (`1` = 1 km, `100` = 100 km)
---              | --
-created.after   | Lower bound for the date the document was created.
-created.before  | Upper bound for the date the document was created.
-modified.after  | Lower bound for document modification date.
-modified.before | Upper bound for document modification date.
---              | --
-with            | Return additional data blocks.
-refs            | Collect specified linked data.
+Name                              | Description
+---------------------             | -----------
```

Comment:
Hmmm, this change seems to have gone missing:
```suggestion
--------------------------------- | -----------
```

Reply:
My mistake :sweat_smile:
Doc build is looking good after this.